extension/src/popup/components/manageNetwork/NetworkForm/index.tsx

```diff
@@ -180,6 +187,13 @@ export const NetworkForm = ({ isEditing }: NetworkFormProps) => {
     }
   };
 
+  const supportsSorobanRpc = (network: string) => {
+    if (network === NETWORK_NAMES.FUTURENET) {
+      return true
+    }
+    return false
+  }
```

Comment:
very NIT: but this would be a little tidier as `const supportsSorobanRpc = (network: string) => network === NETWORK_NAMES.FUTURENET`

Reply:
oh yeah for sure 😄

Reply:
done in 704bac55565f361a39826bbfd8d66bc062ced811